### PR TITLE
feat: add meta open graph and twitter images

### PR DESCRIPTION
**Pull request by Diogomartf**
This PRs adds open graph and twitter meta images.

Following Next js file convention: https://nextjs.org/docs/app/api-reference/file-conventions/metadata/opengraph-image

### CODE REVIEW

**Comment by Diogomartf** [comment]
this PR needs some update on how to get the correct `url` on meta tags.

image

I think it s work out of the box, but let's see if is a bug of next or it has to do with our setup (didn't find on the docs).

https://github.com/vercel/next.js/issues/49406

**Review by berteotti — APPROVED** [review]
Looks good to me! We need to keep in mind that we need to update before launch.

**Comment by Diogomartf** [comment]
Updated to `next.js` 13.4.8 which fixes our issue. I think it's ready to merge!

**Comment by Diogomartf** [comment]
We have a [bug](https://github.com/vercel/next.js/issues/52191) that was introduced on `13.4.8`

It's already [fixed](https://github.com/vercel/next.js/commit/baf3d2fa8a424554f4fe2d6e05dcac1a1ca52c60), we will wait for the next release.

**Comment by Diogomartf** [comment]
Updated to `next@13.4.10` seems like it's wokring good!

@berteotti can we merge this?

File: packages/app/app/layout.tsx
Comment (ElRodrigote) [inline packages/app/app/layout.tsx:13]: Not your change but are these final `..` intended or it's a typo and there should be only a final period?

**Review by ElRodrigote — APPROVED** [review]

File: packages/app/app/layout.tsx
Comment (Diogomartf) [reply packages/app/app/layout.tsx:13]: Not intended. We will iterate on the texts on future updates. This PR will get us going.

**Review by berteotti — APPROVED** [review]
Approved if we review the description later

**Pull request merged**
